README.md

```diff
@@ -9,12 +9,10 @@ We plan to open-source Mojo progressively over time, but it's changing very quic
 We believe that a small, tight-knit group of engineers with a shared vision can move 
 faster than a community effort, so we will continue to incubate it within Modular until 
 it's more complete.  Please see the [Mojo FAQ](https://docs.modular.com/mojo/faq.html)
-for more information about this and other common questions. 
+for more information about this and other common questions. We've opened this repo now
+because we want to gather issues and engage in feedback from Mojo users. 
```

Comment:
```suggestion
because we want to gather issues and invite feedback from Mojo users. 
```

"engage in feedback" just seems odd. "get feedback" would work, too.